lib/workspace.sql

```diff
@@ -34,7 +34,7 @@ FROM
   workspaces AS w
   JOIN variants AS v ON (v.workspace_id = w.id)
   JOIN questions AS q ON (q.id = v.question_id)
-  JOIN pl_courses AS c ON (c.id = q.course_id)
+  JOIN pl_courses AS c ON (c.id = v.course_id)
```

Comment:
I actually don't think we want this change: the `course` returned from this query is used to fetch chunks and read workspace files, which will always be tied to the question itself.

Reply:
ok, reverted. 

What's going to be really fun about merging this PR is that if we mess up, some of those mistakes would show up immediately...but if we make a mistake like this one, then we won't know there is a problem until the question sharing PR is merged and we start having the situation where q.course_id and v.course_id are actually different.

Reply:
I double checked and I agree, the `course` object returned by this query is only used to fetch the question/workspace content.